crypto/nrf_oberon/include/ocrypto_ecdsa_p256.h

```diff
@@ -13,55 +20,72 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
-
-// ecdsa P-256 signature key pair generation
-// pk[64]: the generated public key
-// sk[32]: the private key
-// returns 0 if sk is a legal secret key
+/**@brief ecdsa P-256 signature key pair generation
+ * @param[out]  pk  The generated public key
+ * @param       sk  The Private key
```

Comment:
Is this param in or out? The brief suggests it's out

Reply:
It should be out as well, yes